bird_cloud_gnn/radar_dataset.py

```diff
@@ -116,6 +116,11 @@ def _process_data(self, data, origin=""):
                 )
             ].index
         ).reset_index(drop=True)
+        if len(data) < self.num_neighbours:
+            print(
+                f"Warning: There are not enough points to find {self.num_neighbours} neighbourhood"
```

Comment:
Maybe it is worth adding the `origin` to the message as it makes it easier to identify which file this occurs

Reply:
Added origin